synapse/cortex.py

```diff
@@ -4910,6 +4919,13 @@ def _initStormOpts(self, opts):
         if opts is None:
             opts = {}
 
+        varz = opts.get('vars')
+        if varz is not None:
+            for valu in varz.keys():
+                if not isinstance(valu, str):
+                    mesg = f"Storm var names must be strings (got {valu} of type {type(valu)})"
+                    raise s_exc.BadArg(mesg=mesg)
```

Comment:
Is this something we can do in 2.x.x?

Reply:
In 2.x this check is performed in snap.getStormRuntime